test/keyboard.spec.js

```diff
@@ -146,7 +146,7 @@ module.exports.addTests = function({testRunner, expect, FFOX}) {
           if (event.key === 'l')
             event.preventDefault();
           if (event.key === 'o')
-            Promise.resolve().then(() => event.preventDefault());
```

Comment:
firefox doesn't let you preventDefault after a promise resolve?!

Reply:
apparently, yeah